business_requirement_deliverable/models/business.py

```diff
@@ -227,3 +227,11 @@ def _compute_deliverable_total(self):
         if self.deliverable_lines:
             self.total_revenue = sum(
                 line.price_total for line in self.deliverable_lines)
+            if self.project_id.pricelist_id.currency_id.id != \
```

Comment:
@duanyp1991 I saw the invoice module processing by this method as i have used. I will use the way you provide. Thanks